src/library/scala/collection/mutable/Growable.scala

```diff
@@ -56,10 +56,14 @@ trait Growable[-A] extends Clearable {
    *  @param xs   the IterableOnce producing the elements to $add.
    *  @return  the $coll itself.
    */
+  @nowarn("msg=will most likely never compare equal")
```

Comment:
I have a sudden urge to tweak the error message to "`in all likelihood...`".

Reply:
"should never", perhaps

Reply:
well, except it's not "should never" - this is an example of when it should

Reply:
I feel more like it should be "will in general"

Reply:
also, this needs a category now 😜

Reply:
definitely a cat, I was just highlighting the fragility and the difficulty of ever tweaking thereafter.